bee-hive/bee_hive/step.py

```diff
@@ -12,23 +12,50 @@
 dotenv.load_dotenv()
 from agent import Agent
 
+def eval_expression(expression, prompt):
+    local = {}
+    local["input"] = prompt
+    result = eval(expression, local)
```

Comment:
I think this is fine for now, but we may want to spend some time thinking about how we can mitigate the risks of running / evaluating user code

Reply:
I agree.

Reply:
This implements #126 schema `in _condition`.  The `_sequence` changes will be a separate PR from @developer-gliu.